.github/pull_request_template.md

```diff
@@ -0,0 +1,62 @@
+# Which issue does this PR close?
+
+<!--
+Thanks for opening a pull request!
+If this is your first pull request you can find detailed information on how 
+to contribute here:
+  * [New Contributor's Guide](https://arrow.apache.org/docs/developers/guide/step_by_step/pr_lifecycle.html#reviews-and-merge-of-the-pull-request)
+  * [ContributingOverview](https://arrow.apache.org/docs/developers/overview.html)
```

Comment:
Should they link to stable version or latest development version?

Reply:
Hm probably the Dev version, after all we introduced the warning/disclaimer on the old versions recently:+1: